src/utils/countries-api.js

```diff
@@ -0,0 +1,46 @@
+/* eslint-disable consistent-return */
```

Comment:
- [ ] please kindly avoid using several eslint-disable comments to get around ESlint errors. these ESlint errors are at best showing that you are not adhering to some coding standards or at worst highlighting a possible underlying issue with your code. please kindly address these issues.

Reply:
- [ ] You have **disabled linter check** on your code. Please enable linter check on your code to ensure that your code is robust code and clean as recommended.